src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -112,6 +112,18 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
     append(BronzeTargets.poolsSnapshotTarget)
   )
 
+  lazy private[overwatch] val libsSnapshotModule = Module(1007, "Bronze_Libraries_Snapshot", this)
+  lazy private val appendLibsProcess = ETLDefinition(
+    workspace.getClusterLibraries,
+    append(BronzeTargets.libsSnapshotTarget)
+  )
+
+  lazy private[overwatch] val policiesSnapshotModule = Module(1008, "Bronze_Libraries_Snapshot", this)
```

Comment:
module name should be different, Bronze_Libraries_Snapshot has been used in the above module(1007)